docs/src/index.md

```diff
@@ -2,10 +2,8 @@
 
 This package provides a replacement for [DataArrays.jl](https://github.com/JuliaStats/DataArrays.jl)'s `PooledDataArray` type. Contrary to that type, it supports both non-nullable and nullable arrays (i.e. arrays allowing for missing data), using the [`Null`](https://github.com/JuliaData/Nulls.jl) type. It is also based on a simpler design by only supporting categorical data, which allows offering more specialized features (like ordering of categories). See the [IndirectArrays.jl](https://github.com/JuliaArrays/IndirectArrays.jl) package for a simpler array type storing data with a small number of values.
 
-The package provides the `CategoricalArray` type designed to hold categorical data (either unordered/nominal or ordered/ordinal) efficiently and conveniently. `CategoricalArray{T}` holds values of type `T`. `CategoricalArray{Union{Null, T}}`, either holds values of type `T`, or missing values (represented as `null`, of the [`Null`](https://github.com/JuliaData/Nulls.jl) type). When indexed, `CategoricalArray{T}` returns values of type `CategoricalValue{T}` rather than of type `T` (or `null` for missing values).
+The package provides the `CategoricalArray` type designed to hold categorical data (either unordered/nominal or ordered/ordinal) efficiently and conveniently. `CategoricalArray{T}` holds values of type `T`. `CategoricalArray{Union{Null, T}}` variant can also contain missing values (represented as `null`, of the [`Null`](https://github.com/JuliaData/Nulls.jl) type). When indexed, `CategoricalArray{T}` returns objects of type `CategoricalValue{T}` rather than the original values of type `T`. `CategoricalValue` type is a simple wrapper around the categorical levels that provides very efficient retrieval and comparison of actual values.
```

Comment:
Add "The" at the beginning of sentences where appropriate. "Allows" was better than "provides" IMHO, but maybe we can find another term.